app/definition/IVisitor.ts

```diff
@@ -0,0 +1,24 @@
+export interface IVisitorEmail {
+	address: string;
+}
+
+export interface IVisitorPhone {
+	phoneNumber: string;
+}
+
+export interface IVisitor {
+	_id?: string;
+	token: string;
+	username: string;
+	updatedAt?: Date;
+	name: string;
+	department?: string;
+	phone?: Array<IVisitorPhone>;
+	visitorEmails?: Array<IVisitorEmail>;
```

Comment:
It's not wrong, but just for the sake of consistency, we should use the same you've used on the other file: `string[]`
```suggestion
	phone?: IVisitorPhone[];
	visitorEmails?: IVisitorEmail[];
```